examples/modular-drivers/README.md

```diff
@@ -108,28 +110,34 @@ platform. The generated project has the form of a library crate, where
 `src/lib.rs` defines an initially empty implementation of the
 well-known Micro-RDK module entry point `register_models`:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/src/lib.rs#L3-L5
+``` rust
+use micro_rdk::common::registry::{ComponentRegistry, RegistryError};
+
+pub fn register_models(_registry: &mut ComponentRegistry) -> anyhow::Result<(), RegistryError> {
+    Ok(())
+}
+```
 
 The generated project also includes a `package.metadata` section in
 its `Cargo.toml` which identifies the library crate as being a
 Micro-RDK module:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/Cargo.toml#L12-L13
+https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/Cargo.toml#L18-19
 
 A subsequent commit adds definitions of the
-[FreeHeapSensor](https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/wifi_rssi_sensor.rs)
+[FreeHeapSensor](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/src/free_heap_sensor.rs)
```

Comment:
Is there a way to make these links relative so the `github.com/viamrobotics/micro-rdk` part can be removed? Does
`https://../../blob/...` work? 

The answer may be https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files